public/app/features/manage-dashboards/partials/dashboard_list.html

```diff
@@ -1,7 +1,8 @@
 <page-header model="ctrl.navModel"></page-header>
 
 <div class="page-container page-body">
-  <manage-dashboards />
+  <!-- Need to pass the props, otherwise React component doesn't render -->
+  <manage-dashboards folder-id="undefined" folder-uid="undefined" />
```

Comment:
looks like we can make this a full react container instead now? or are there still things in the DashboardListCtrl  ?

Reply:
Good idea, do I need to add the component to the router using `<react-container />` for that?